services/gtfs-rt-archiver-v3/gtfs_rt_archiver_v3/consumer.py

```diff
@@ -15,6 +16,8 @@
 
 from .tasks import RTFetchException, huey
 
+compiled_regex = re.compile("0x[0-9a-fA-F]+")
```

Comment:
I don't know rigorously how memory addresses are allocated to HTTPSConnection objects, but in practice it seems that the object reference always ends up `0x7f...` in our environment. I considered that we could tighten this filter to reflect that, but not knowing whether there's a hard and fast rule preventing spillover into 0x8... space or 0x7e... space, I don't actually advise doing so unless you have supporting information.